confgenerator/confgenerator_test.go

```diff
@@ -89,6 +89,7 @@ func TestGenerateConfsWithValidInput(t *testing.T) {
 	t.Parallel()
 	filepathJoin = testJoin
 	for _, platform := range platforms {
+		platform := platform // create a new 'platform'
```

Comment:
This comment isn't particularly helpful. You could use the FAQ URL, or just omit the comment, similarly to all the other places this pattern occurs.